scripts/dotnet-cli-build/Utils/BuildVersion.cs

```diff
@@ -13,6 +13,7 @@ public class BuildVersion
         public string VersionSuffix => $"{ReleaseSuffix}-{CommitCountString}";
         public string NuGetVersion => $"{Major}.{Minor}.{Patch}-{VersionSuffix}";
         public string NetCoreAppVersion => $"{Major}.{Minor}.{Patch}-rc2-3{CommitCountString}";
+        public string HostVersion => $"{Major}.{Minor}.1-rc2-{CommitCountString}-00";
```

Comment:
Why is the host version different from the CLI version? You will have to make changes in the installers too to use this version.